### PR TITLE
Fix typos/formatting in tutorials

### PR DESCRIPTION
**Pull request by AlexBass05**
Thanks for adding the tutorials! They are a great way to get started with PROJ.

While reading through them I spotted a couple of typos and formatting inconsistencies.

### CODE REVIEW

File: docs/source/tutorials/EUREF2019/answers.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Projections.1: UTM
     3.      +proj=utm +zone=35 +ellps=intl
 
 
-Projections2.: LCC
+Projections.2: LCC
```
Comment (kbevers) [inline docs/source/tutorials/EUREF2019/answers.rst]: Shouldn't the dot just be removed all together?

Comment (AlexBass05) [reply docs/source/tutorials/EUREF2019/answers.rst]: I don't know... maybe? The format with the dot is used above in `Projection.1` and below in `Projection.3`.
This change is only meant to align the formatting.

Comment (kbevers) [reply docs/source/tutorials/EUREF2019/answers.rst]: I think they were supposed to reflect the naming of the gie files. Så `projection1` etc would be correct

Comment (AlexBass05) [reply docs/source/tutorials/EUREF2019/answers.rst]: I see. Makes sense. I removed the dots.

**Pull request merged**